src/caughtup/caughtUpReducer.js

```diff
@@ -32,10 +32,14 @@ export default (
   action: PerAccountApplicableAction,
 ): CaughtUpState => {
   switch (action.type) {
-    case REGISTER_COMPLETE:
     case RESET_ACCOUNT_DATA:
       return initialState;
 
+    // Reset because `fetching` is server-data metadata, and we're resetting
+    // the server data it's supposed to apply to.
+    case REGISTER_COMPLETE:
+      return initialState;
```

Comment:
Should this say "`caughtUp`" instead of "`fetching`"?

Also it'd be good to spell out what that other server data is that we're resetting that causes us to reset this: namely, `narrows`.